.sync.yml

```diff
@@ -0,0 +1,26 @@
+---
+.travis.yml:
+  extras:
+    - rvm: 1.9.3
+      env: PUPPET_GEM_VERSION="3.2.1"
+    - rvm: 2.0.0
+      env: PUPPET_GEM_VERSION="3.2.1"
```

Comment:
3.2.1 - Where did you get this number? Also, it was only 1.9.3 until 3.7.0 (tribal knowledge, apologies).